src/Github/ChangelogGenerator.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\AutomaticReleases\Github;
+
+use Doctrine\AutomaticReleases\Git\Value\SemVerVersion;
+use Doctrine\AutomaticReleases\Github\Value\RepositoryName;
+
+interface ChangelogGenerator
```

Comment:
Since this is now a function: `GenerateChangelog`

Reply:
Renamed.